orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/pipeline/UpdatePipelineTemplateStage.java

```diff
@@ -35,4 +60,61 @@
     builder
       .withTask("updatePipelineTemplate", UpdatePipelineTemplateTask.class);
   }
+
+  @Nonnull
+  @Override
+  public <T extends Execution<T>> List<Stage<T>> aroundStages(@Nonnull Stage<T> stage) {
+    if (front50Service == null) {
+      return Collections.emptyList();
+    }
+
+    if (!stage.getContext().containsKey("pipelineTemplate")) {
+      throw new IllegalArgumentException("Missing required task parameter (pipelineTemplate)");
+    }
+
+    if (!(stage.getContext().get("pipelineTemplate") instanceof String)) {
+      throw new IllegalArgumentException("'pipelineTemplate' context key must be a base64-encoded string: Ensure you're on the most recent version of gate");
+    }
+
+    PipelineTemplate pipelineTemplate = stage.decodeBase64(
+      "/pipelineTemplate",
+      PipelineTemplate.class,
+      pipelineTemplateObjectMapper
+    );
+
+    if (!pipelineTemplate.getConfiguration().hasInheritableConfigs()) {
+      // Pipeline templates that don't have inheritable configurations don't need to cascade save child pipelines.
```

Comment:
Isn't this something you would want to propagate? e.g., if the change is that the template no longer has triggers, you'll want to remove them from its dependent pipelines.